tests/test_manifest.py

```diff
@@ -1,26 +1,124 @@
 import logging
 import pytest
+from click.testing import CliRunner
```

Comment:
Guessing `CliRunner` is not required at the moment?

Reply:
Very perceptive of you! A previous version of this test suite used the `get_manifest()` CLI function, but I ended up "rebuilding" that function here using a combination of fixtures.